app/views/places/show.html.erb

```diff
@@ -41,31 +41,20 @@ function initMap() {
         src="https://maps.googleapis.com/maps/api/js?signed_in=true&callback=initMap"></script>
         <div id="map"></div>
 
- 
-<div class="clear"></div>
-<%= simple_form_for @comment, url: place_comments_path(@place) do |f| %>
-  <%= f.input :message %>
-  <%= f.input :rating, as: :select, collection: Comment::RATINGS %>
-  <%= f.submit "Submit Your Comment", class: 'btn btn-primary' %>
-<% end %>
 
 <div class="clear"></div>
-<%= simple_form_for @photo, url: '#' do |f| %>
+<%= simple_form_for @photo, url: place_photos_path(@place) do |f| %>
   <%= f.input :caption %>
-  <%= f.submit "Add a Caption", class: 'btn btn-primary' %>
+  <%= f.submit "Submit", class: 'btn btn-primary' %>
```

Comment:
I changed the text of the submit button, because eventually photos will be uploaded here.